### ISSUE TITLE
Testing NextJS apps fails

### ISSUE THREAD

**Issue by rarmatei** [nrwl/nx#4281]
## Current Behavior

Running tests for a NextJS app bootstrapped inside an Nx React workspace fails on Node 12.

![image](https://user-images.githubusercontent.com/5975076/102082310-a80d3e00-3e09-11eb-9d16-e57b61029e51.png)

## Expected Behavior

They should not fail.

## Steps to Reproduce

Make sure you're running Node 12:

```bash
nvm use 12.0.0
npx create-nx-workspace@latest test1 --preset=react --style=scss --appName=blog --nxCloud=false
cd test1
yarn add @nrwl/next
yarn nx generate @nrwl/next:app ecommerce --style=scss
yarn nx test ecommerce
```

### Failure Logs

```bash
> yarn nx test ecommerce
yarn run v1.22.5
$ nx test ecommerce

> nx run ecommerce:test
 FAIL   ecommerce  apps/ecommerce/specs/index.spec.tsx
  ● Test suite failed to run

    Cannot find module './babel-jest.config.json'
    Require stack:
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest-config/node_modules/@babel/core/lib/config/files/configuration.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest-config/node_modules/@babel/core/lib/config/files/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest-config/node_modules/@babel/core/lib/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest-config/node_modules/babel-jest/build/loadBabelConfig.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest-config/node_modules/babel-jest/build/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@jest/transform/build/ScriptTransformer.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@jest/transform/build/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest-runtime/build/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@jest/core/build/cli/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@jest/core/build/jest.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/jest/build/jest.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@nrwl/jest/src/builders/jest/jest.impl.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@nrwl/tao/src/shared/workspace.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@nrwl/tao/src/commands/run.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@nrwl/tao/index.js
    - /Users/raresmatei/Projects/nrwl/tmp/Dec/test1/node_modules/@nrwl/cli/lib/run-cli.js

      at loadConfig (../../node_modules/jest-config/node_modules/@babel/core/lib/config/files/configuration.js:157:5)
          at loadConfig.next (<anonymous>)
          at buildRootChain.next (<anonymous>)
          at loadPrivatePartialConfig.next (<anonymous>)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.759 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### Environment

```
>  NX  Report complete - copy this into the issue template

  Node : 12.0.0
  OS   : darwin x64
  yarn : 1.22.5

  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.2
  @nrwl/cypress : 11.0.2
  @nrwl/devkit : 11.0.2
  @nrwl/eslint-plugin-nx : 11.0.2
  @nrwl/express : Not Found
  @nrwl/jest : 11.0.2
  @nrwl/linter : 11.0.2
  @nrwl/nest : Not Found
  @nrwl/next : 11.0.2
  @nrwl/node : Not Found
  @nrwl/react : 11.0.2
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.2
  @nrwl/web : 11.0.2
  @nrwl/workspace : 11.0.2
  typescript : 4.0.5
```

**Comment by bekos** [comment]
@rarmatei can you check if this is still an issue with the latest Nx? I'm not able to reproduce it 🤔

**Comment by rarmatei** [comment]
> @rarmatei can you check if this is still an issue with the latest Nx? I'm not able to reproduce it 🤔

It still happens for me on latest unfortunately - are you definitely using Node v12.0.0?

**Comment by rarmatei** [comment]
I tried on Node v12.9 and it works okay (it might work with earlier versions as well) - so I guess we can close this 👍

**Comment by bekos** [comment]
Yup, this seems to be a Node issue happening only for Yarn, seems to be fixed at least on v12.3.
